kola/tests/misc/selinux.go

```diff
@@ -25,44 +27,188 @@ func init() {
 		ClusterSize: 1,
 		Name:        "coreos.selinux.enforce",
 		Flags:       []register.Flag{register.NoEnableSelinux},
+		Distros:     []string{"cl"},
+	})
+
+	// TODO:  fall back to SelinuxEnforce when PATH on RHCOS is sane
+	// See https://github.com/openshift/os/issues/191
+	register.Register(&register.Test{
+		Run:         SelinuxEnforceRhcos,
+		ClusterSize: 1,
+		Name:        "coreos.selinux.enforce.rhcos",
```

Comment:
The pattern we've been using is to replace the `coreos` with `rhcos`

Reply:
Noted.

This might be moot very soon, now that we have SELinux support enabled in RHCOS and the PATH should be sane.

I'll retry this test once we have a new cloud image built with the SELinux support + PATH changes intact.

Reply:
selinux support is now enabled in the latest composes.